SpatialGDK/Setup.bat

```diff
@@ -7,7 +7,7 @@ pushd "%~dp0"
 call :MarkStartOfBlock "%~0"
```

Comment:
Maybe this should be in the root directory and not in SpatialGDK. I think it's a little strange having it in a subfolder when there is SpatialGDK and SpatialGDKEditorToolbar.

Reply:
I see your point but to me it feels strange to have to go to a sub-folder to actually 'install' the plugin if you know what I mean.